### PR TITLE
[New] Project subsection

### LINKED ISSUE #673: Make "Project" a subsection of Experience
**Is your feature request related to a problem? Please describe.**
Projects shall be associated with specific jobs for ease of reference and indication of dates/role.

#435 

**Describe the solution you'd like**
For each Experience (Experience: 1):
In the Edit form, below the Description box add a field and box called Projects where Users can add their list of projects 
Project does not need it's own privacy setting
Project shall be a searchable item

image

image

### PR DESCRIPTION
**Pull request by mo-5**
#### Changes introduced
Added project as a subsection of experience.

#### Related issue(s)
closes #673

#### Screenshots (if aplicable)
image

image

#### Checklist
If the database has been modified:
- [x] Update database diagram 
- [x] Create new migration 

If API endpoints has been modified:
- [ ] Update backend documentation 
- [ ] Create/Update validators for the API endpoints 
- [x] The API endpoints are properly secured with keycloak 

If the UI has been modified:
- [x] It is translated in both language (with no hard coded text) 
- [ ] Has been tested on IE
- [x] The modifications are tab friendly
- [x] It is accessible

### CODE REVIEW

**Review by BenJeau — COMMENTED** [review]
Looks good, one small comment - Just need to test it locally

File: services/backend/package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
   "author": "I-Talent team",
   "license": "MIT",
   "dependencies": {
-    "@prisma/client": "2.5.0",
-    "axios": "^0.19.2",
+    "@prisma/client": "2.5.1",
+    "axios": "^0.20.0",
```
Comment (dynamic11) [inline services/backend/package.json]: Maybe we should do this upgrade in a separate PR in case it starts causing issues (it will be easier to isolate)

**Review by dynamic11 — CHANGES_REQUESTED** [review]

**Comment by BenJeau** [comment]
Aside from @dynamic11 comment (prisma 2.6 just got release!), the code seems to work fine :100:, the only change I would do is making the tags gray instead of green (for projects)

**Comment by mo-5** [comment]
Should now be resolved

**Comment by BenJeau** [comment]
I'll try it right now

**Comment by BenJeau** [comment]
Two small comments

1. I think the color of the tag is a bit light, I think it should be the same as the others, #727272
image

2. The tags in the forms is not the same styles as the others in forms (I think you just need to add a className to the input)
image
image

**Comment by mo-5** [comment]
Shape and colour have been changed:
1.
image
2.
image

**Review by BenJeau — APPROVED** [review]

**Review by dynamic11 — APPROVED** [review]

**Pull request merged**